app/controllers/api/v2/roles_controller.rb

```diff
@@ -23,6 +23,8 @@ def show
       def_param_group :role do
         param :role, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :organization_ids, Array
```

Comment:
actually not, the description is different, roles are not scoped by taxonomy but I took the condition from taxonomies param group